mastracode/src/web/.env.example

```
@@ -0,0 +1,47 @@
+# vars for a feature are absent, that feature is a no-op and the web app
+# behaves exactly as before.
+
+# ---------------------------------------------------------------------------
+# WorkOS auth
+# When WORKOS_API_KEY and WORKOS_CLIENT_ID are set, every route requires a
+# signed-in user (hosted login + encrypted session). This is a prerequisite
+# for the GitHub projects feature below.
+# ---------------------------------------------------------------------------
+WORKOS_API_KEY=
+WORKOS_CLIENT_ID=
+WORKOS_REDIRECT_URI=https://your-host/auth/callback   # optional
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)